cspell.json

```diff
@@ -114,7 +114,8 @@
         "test_collections/matter/sdk_tests/sdk_checkout",
         "test_collections/matter/python_tests/docs/common_test_failures/",
         "sdk_patch",
-        ".devcontainer"
+        ".devcontainer",
+        "test_collections/matter/sdk_tests/*.json"
```

Comment:
Should the json file for non-custom test cases also be ignored?

Reply:
The custom json file is indicated at .ignore file, this is already the non-custom

Reply:
I got confused with the .gitignore file, sorry!